README.md

```diff
@@ -1,2 +1,226 @@
+        steps:
+          - name: Set Assignee and Reviewer when Checks Ready
+            uses: ChrisCarini/set-assignee-reviewer-when-ready@latest
+            with:
+              assignees: 'ChrisCarini'
+              reviewers: 'ChrisCarini'
+              waitSeconds: 30
+              token: ${{ secrets.REPO_SCOPE_GITHUB_TOKEN }}
+    ```
+
+## ⚙️ Options
+
+This GitHub Action exposes several input options.
+
+### Option Table
+
+|            Input            | Description                                                                                                                 |     Usage      |                Default                 |
+|:---------------------------:|:----------------------------------------------------------------------------------------------------------------------------|:--------------:|:--------------------------------------:|
+|            `token`          | The GitHub token to use for making API requests.                                                                            |   *Required*   |                  N/A                   |
+|    `acceptableConclusions`  | The acceptable conclusion for each check run that has completed. Defaults to all known conclusions.                         |   *Optional*   | `success, neutral, cancelled, skipped` |
+|   `unacceptableConclusions` | The unacceptable conclusion for each check run that has completed. Defaults to no unacceptable conclusions.                 |   *Optional*   | `failure, timed_out, action_required`  |
+|          `assignees`        | List of assignees to assign the PR to. Defaults to no one.                                                                  | *Required* (*) |                _No One_                |
+|          `reviewers`        | List of reviewers to request reviews on the PR. Defaults to no one.                                                         | *Required* (*) |                _No One_                |
+|     `requiredChecksOnly`    | Only consider `required checks`. Defaults to true.                                                                          |   *Optional*   |                 `true`                 |
+|         `waitSeconds`       | If all checks having acceptable conclusions, wait N seconds before requesting reviewers on the PR. Defaults to 10 minutes.  |   *Optional*   |                 `600`                  |
+
+`(*)` = You can set one (ie, either `assignees` or `reviewers`), or both of these inputs. Setting neither of these inputs doesn't do anything.
+
+### Example using all options
+
+An example using all the available options is below:
+
+```yaml
+    - name: Set Assignee and Reviewer when Checks Ready
+      uses: ChrisCarini/set-assignee-reviewer-when-ready@latest
+      with:
+        acceptableConclusions: 'success'    # The only 'acceptable conclusion' to a check run is `success`.
+        unacceptableConclusions: 'failure'  # The only 'unacceptable conclusion' to a check run is `failure`. 
```

Comment:
The term conclusion, while correct, really only makes sense to people who author more complicated actions. Is there something else you could think to call this?

Reply:
Before learning it was called 'conclusion' in GH parlance, I was inclined to think one of the below:

- `state` (eg. `acceptableStates`)
- `result` (eg. `acceptableResults`)
- `status` (eg. `acceptableStatuses`)

I have no strong preference here, but I _totally_ agree that 'conclusion' was both not intuitive to me, and a pain to type / read repeatedly throughout developing this...

Let me know if you have any preference, and I can update accordingly!